### PR TITLE
Change to alpine to save 500+ MB image size

### PR DESCRIPTION
**Pull request by zhouzhuojie**
Everyone loves ArchLinux, but we probably don't need everything in ArchLinux.

```
# before
codeflow              v004                1d174e5dddca        34 minutes ago       1.88GB

# after
codeflow              v004                f018023e1caf        34 seconds ago      1.33GB
```

Also, tested in dev

image

### CODE REVIEW

File: server/configs/codeflow.yml [hunk 1]
```diff
@@ -65,14 +65,14 @@ plugins:
     registry_password: ""
     registry_user_email: ""
     docker_host: "unix:///var/run/docker.sock"
-    workdir: "/tmp/docker_build"   
+    workdir: "/tmp/docker_build"
   kubedeploy:
```
Comment (zhouzhuojie) [inline server/configs/codeflow.yml]: lol, IMPORTANT spaces fixed

**Comment by sasso** [comment]
Reason for using ARCH is github.com/libgit2/git2go **v25**. All other distros at that time(and probably still) used v24 that was crashing in some cases when syncing repositories. Is this packed with v25?

**Comment by zhouzhuojie** [comment]
Yes, `0.25.1-r2`
https://pkgs.alpinelinux.org/package/edge/main/x86/libgit2-dev

**Comment by sasso** [comment]
👌

**Comment by sasso** [comment]
Thank you @zhouzhuojie for contributing!

**Pull request merged**